### ISSUE TITLE
FormBuilderDateTimePicker: initial value not set

### ISSUE THREAD

**Issue by thirteenthstep** [danvick/flutter_form_builder#425]
FormBuilderDateTimePicker does not seem to apply the initial value at any point.

**Comment by thirteenthstep** [comment]
Sry I forgot to mention this is affecting version 4.0.0-beta.1.
Following examples are both working in 3.13.3, but not in 4.0.0-beta.1 - there is no date set:

````
FormBuilder(
  child: FormBuilderDateTimePicker(
    name: 'foo',
    initialValue: DateTime.now(),
    ),
 ),
````

````
FormBuilder(
  initialValue: {
    'foo': DateTime.now(),
  },
  child: FormBuilderDateTimePicker(
    name: 'foo',
  ),
),
````

**Comment by danvick** [comment]
Noted. Thanks for bringing this to my attention. I'll have a look and revert ASAP

**Comment by danvick** [comment]
Should be fixed in v4.0.0-beta.2

**Comment by fpicoitoj** [comment]
Hello @danvick @thirteenthstep 
Is there any way to go around this until v4 drops? I've also stumbled upon initialValue not being set properly display-wise.
